docs/api.rst

```diff
@@ -0,0 +1,37 @@
+===
+API
+===
+
+Similarly to Croud, we offer API to allow programmatic access to our Cloud
+products. API is operated using a key and a secret. These can be created in
```

Comment:
I would perhaps phrase it as "The API can be accessed by generating a key and secret in your account page"